common/src/main/java/mod/chiselsandbits/item/ChiseledBlockItem.java

```diff
@@ -175,31 +177,33 @@ public void appendHoverText(
     }
 
     @Override
-    public boolean canPlace(final ItemStack heldStack, final Player playerEntity, final BlockHitResult blockRayTraceResult)
+    public PlacementResult getPlacementResult(final ItemStack heldStack, final Player playerEntity, final BlockHitResult blockRayTraceResult)
     {
         final IAreaAccessor source = this.createItemStack(heldStack);
-        final Vec3 target = getTargetedBlockPos(heldStack, playerEntity, blockRayTraceResult);
+        final Vec3 target = getTargetedPosition(heldStack, playerEntity, blockRayTraceResult);
         final IWorldAreaMutator areaMutator = IMutatorFactory.getInstance().covering(
           playerEntity.level,
           target,
           target.add(1,1,1));
         final IMultiStateSnapshot attemptTarget = areaMutator.createSnapshot();
 
-        return source.stream()
-          .allMatch(stateEntryInfo -> {
-              try
-              {
-                  attemptTarget.setInAreaTarget(
-                    stateEntryInfo.getBlockInformation(),
-                    stateEntryInfo.getStartPoint());
-
-                  return true;
-              }
-              catch (SpaceOccupiedException exception)
-              {
-                  return false;
-              }
-          });
+        final boolean noSpaceOccupied = source.stream()
+                .allMatch(stateEntryInfo -> {
+                    try {
+                        attemptTarget.setInAreaTarget(
+                                stateEntryInfo.getBlockInformation(),
+                                stateEntryInfo.getStartPoint());
+
+                        return true;
+                    } catch (SpaceOccupiedException exception) {
+                        return false;
+                    }
+                });
+        return noSpaceOccupied
+                ? PlacementResult.success()
+                : PlacementResult.failure(
+                        IClientConfiguration.getInstance().getNotFittingPatternPlacementColor().get(),
```

Comment:
Again this is not a good idea......
The client configuration might not be loaded on the server side and some platforms might throw errors if still accessed.